doc/build/faq/sessions.rst

```diff
@@ -413,18 +413,18 @@ and expires all state::
 
     Session.commit()  # expires all attributes
 
-    foo_7 = Session.query(Foo).get(7)
+    foo_7 = session.get(Foo, 7)
 
     assert o.foo is foo_7  # o.foo lazyloads on access
 
 A more minimal operation is to expire the attribute individually - this can
 be performed for any :term:`persistent` object using :meth:`.Session.expire`::
 
-    o = Session.query(SomeClass).first()
+    o = session.scalars(select(SomeClass)).first()
```

Comment:
**mike bayer** (zzzeek) wrote:

.limit(1)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3729

Reply:
Changed